### PR TITLE
Update range

### PR DESCRIPTION
**Pull request by hrtnkpn**
### What's new?

In the mixin of the range component, it is written that you need to show the `div.range_gradient` if type is alpha. Only you can simplify everything. Each range class has an `.is-{type}`. Plus, `.is-a` already had styles. I removed the `div.range_gradient` element and just add a pseudo-element exclusively from `.is-a`. After I changed the logic of the mixin and make a canvas render when `type !== alpha`. I think it's easier and more logical

### CODE REVIEW

File: view/range/index.css [hunk 1]
```diff
@@ -12,11 +12,17 @@ body {
   &.is-a {
     background: var(--opacity-pattern);
     border: none;
+
+    &::before {
```
Comment (ai) [inline view/range/index.css:16]: According to my code style, `::before` rule should be at separated line (not nested).

I am thinking of `::before` as an element.

**Comment by hrtnkpn** [comment]
I have corrected. Look, did I understand you correctly?

File: view/range/index.css [hunk 2]
```diff
@@ -13,10 +13,16 @@ body {
     background: var(--opacity-pattern);
     border: none;
   }
+
+  &.is-a::before {
```
Comment (ai) [inline view/range/index.css:17]: This rules should be converted to `.range.is-a::before` too

Comment (hrtnkpn) [reply view/range/index.css:17]: I don't understand what's wrong?

**Comment by ai** [comment]
Here is how I fixed it (2 properties is too short to share them) ff70363

**Comment by hrtnkpn** [comment]
Okay. Thank you!

**Pull request merged**